src/components/accordion/accordion.e2e.ts

```diff
@@ -195,4 +195,33 @@ describe("calcite-accordion", () => {
     expect(await item2Content.isVisible()).toBe(true);
     expect(await item3Content.isVisible()).toBe(false);
   });
+
+  it("renders multiple expanded items when selection mode changes from single to multiple", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`
+    <calcite-accordion selection-mode="single">
+    ${accordionContent}
+    </calcite-accordion>`);
+    const element = await page.find("calcite-accordion");
+    expect(element).toEqualAttribute("selection-mode", "single");
+    element.setAttribute("selection-mode", "multiple");
+    await page.waitForChanges();
+    const item1 = await element.find("calcite-accordion-item[id='1']");
```

Comment:
Suggestion: `const [item1, item2, item3] = await element.findAll("calcite-accordion-item");`. Totally up to you. 😄

Reply:
Clean 🧹 . applied this to other tests too.